components/back-top/index.tsx

```diff
@@ -40,10 +40,6 @@ export interface BackTopProps {
 }
 
 export default class BackTop extends React.Component<BackTopProps, any> {
-  static defaultProps = {
-    visibilityHeight: 400,
-  };
```

Comment:
does this syntax will auto bind `this.props.visibilityHeight` ? I understand it‘s just something like `default parameter`.
so what if call `this.props.visibilityHeight` in constructor or lifeCycle and other function?  I guess it will be `undefined`, then this will cause problem